tensorflow/python/keras/backend.py

```diff
@@ -2898,7 +2904,11 @@ def __call__(self, inputs):
         session != self._session):
       self._make_callable(feed_arrays, feed_symbols, symbol_vals, session)
 
-    fetched = self._callable_fn(*array_vals)
+    if self.run_metadata:
+      fetched = self._callable_fn(*array_vals,
+                                  run_metadata=self.run_metadata)
```

Comment:
Is it possible to always use lines 2908-2099? I'm guessing (and not sure) that if `self.run_metadata` is `None`, it'll be handled correctly?

Reply:
Thank you, I think so.